activiti-cloud-services-runtime-bundle/activiti-cloud-services-core/src/main/resources/image/na.svg

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!-- Generator: Adobe Illustrator 18.1.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->

```

Comment:
@constantin-ciobotaru this might cause problems. It might be good to find a Creative Commons licensed SVG.

Reply:
Ups, I didn't realize it :(
Thanks.